src/d3d9/d3d9_device.cpp

```diff
@@ -5091,6 +5117,159 @@ namespace dxvk {
     return D3D_OK;
   }
 
+  
+
+  void D3D9DeviceEx::UploadDynamicSysmemBuffers(
```

Comment:
Do we want to mark this as inline?

Reply:
We could do that but we don't mark PrepareDraw as inline either.

Reply:
Ehh fair enough